cadastre/cadastre_import.py

```diff
@@ -461,6 +461,10 @@ def get_available_majic_files(self) -> tuple[dict, dict]:
                         file_path = os.path.join(root, file_sub_path)
                         maj_list.append(file_path)
 
+                        # ignore PDF files
+                        if file_path.endswith(".pdf") or file_path.endswith(".PDF"):
```

Comment:
```suggestion
                        if file_path.lower().endswith(".pdf"):
```

Or `if file_path.endswith((".pdf", ".PDF)):` is also possible, quite convenient to check multiple extensions.